game-core/src/main/java/games/strategy/triplea/attachments/CanalAttachment.java

```diff
@@ -146,10 +155,6 @@ public void setExcludedUnits(final HashSet<UnitType> value) {
     return m_excludedUnits;
   }
 
-  public void clearExcludedUnits() {
```

Comment:
Some of the tests in `ValidateAttachmentsTest` are failing because

> Class games.strategy.triplea.attachments.CanalAttachment doesn't have a clear method for 'adder' property excludedUnits